config.sample.yaml

```diff
@@ -556,6 +556,9 @@ ircService:
     # how much time needs to pass between the reply and the original message to switch to the long format
     shortReplyTresholdSeconds: 300
 
+  # Maximum number of montly active users, beyond which the bridge gets blocked (both ways)
+  # RMAUlimit: 100
```

Comment:
Needs a JSON schema change too :stuck_out_tongue:

Reply:
Done in 26505376 :)